### ISSUE TITLE
Missing Queue for Host 1.6k errors in 3 days from AppInsights on WebApp

### ISSUE THREAD

**Issue by OzBob** [Azure/azure-webjobs-sdk#2528]
I have running webjobs, continuously for months, and are continuing to run now, with no specific failures logged in their output, on the WebJob SDK kudu webpages.

BUT

Application Insights have reported 1,640 errors from the WebJob Host.

The error was: 404 when requesting:
https://{name}.queue.core.windows.net:443/azure-webjobs-host-abe5fb6ff9ec4c478f2a7f7d696f7dbe?comp=metadata

The queue does not exist.

Why would the WebJob Host suddenly request 1600+ times?

The last message of this type was received:  24/06/2020, 3:26:13 AM GMT+8. No further errors are being received.

What should I do to prevent this from hapenning again?

#### Repro steps

Provide the steps required to reproduce the problem

1. Run continuous WebJob (SDKs: Microsoft.Azure.WebJobs version=2.3.0, WindowsAzure.Storage version=9.3.3) in Azure WebApp for months.

2. Suddenly get 1000s of errors looking for a missing host queue

3. Change nothing 

4. Errors stop

#### Expected behavior

no errors.

#### Actual behavior

as per above

#### Known workarounds

na

#### Related information 

Provide any related information 

* Package version: Microsoft.Azure.WebJobs version=2.3.0, WindowsAzure.Storage version=9.3.3

**Comment by OzBob** [comment]
Workaround: read applicationinsights logs, and find the name of the 'missing' queue and manually create it.

**Comment by OzBob** [comment]
I found the same error on my UAT system. I did not manually create the webjob, this time i restarted the webapp, and the error stopped being logged in Application Insights.

So, my conclusion is that the what ever Microsoft/azure team did to the Host system to cause a ‘Health Event’, did not restart the webapp and caused some webjob ‘host’ to start trying to connect to a queue that had not been created, as the storage part of the system that the webjob host depended on had not completed it’s restart.
